ui/pl.json

```diff
@@ -0,0 +1,24 @@
+{
+  "building-delete": "Usuń",
+  "building-move": "Przesuń",
+  "contact_completed": "Kontrakt został wypełniony.",
+  "contact_unlocked": "Nowy kontrakt został odblokowany.",
+  "equipment_unknown": "Błąd: Nieznany ekwipunek.", //Review and context needed here!
```

Comment:
This is an error sent from the server when it doesn't find the equipment you're trying to move or delete. And the translation seems correct to me.